notebooks/community/sdk/sdk_private_endpoint.ipynb

```diff
@@ -0,0 +1,1015 @@
+{
```

Comment:
<p>Use the preferred form for links to webdocs:</p><p> </p><p>Learn more about [title](link).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>